packages/common/src/content/_internal/fetchEditableContentEnrichments.ts

```diff
@@ -0,0 +1,76 @@
+import { IItem } from "@esri/arcgis-rest-portal";
+import {
+  EditableContentEnrichment,
+  IHubEditableContentEnrichments,
+  ItemOrServerEnrichment,
+  fetchItemEnrichments,
+} from "../../items/_enrichments";
+import { isService } from "../../resources/_internal/_validate-url-helpers";
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
+import { IHubRequestOptions } from "../../types";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { fetchItemScheduleEnrichment } from "./fetchItemScheduleEnrichment";
+
+/**
+ * @private
+ *
+ * Fetches the enrichments for a content item to be converted into an IHubEditableContent object.
+ * If no enrichment keys are provided, the default enrichments will be fetched. Default enrichments
+ * vary by item type.
+ *
+ * @param item item to fetch enrichments for
+ * @param requestOptions
+ * @param enrichments optional override for the enrichments to fetch
+ * @returns a hash of enrichments
+ */
+export async function fetchEditableContentEnrichments(
+  item: IItem,
+  requestOptions: IRequestOptions,
+  enrichments?: EditableContentEnrichment[]
+): Promise<IHubEditableContentEnrichments> {
+  const result: IHubEditableContentEnrichments = {};
+
+  if (!enrichments) {
+    enrichments = getDefaultEnrichmentKeys(item);
+  }
+
+  // Leverage the existing fetchItemEnrichments() function to fetch supported enrichments
+  const itemOrServerEnrichmentKeys = enrichments.filter(
+    (e) => e !== "schedule"
+  );
```

Comment:
Are there future enrichments that would be unsupported?  Just wondering if we need to comment why some enrichments are unsupported and maybe move to array, something like:

```suggestion
  // The following item enrichments are not supported because...
  const unsupportedEnrichments = ["schedule"]
  const itemOrServerEnrichmentKeys = enrichments.filter(
    (e) => !unsupportedEnrichments.includes(e)
  );
```

Reply:
Good call Josh! I don't foresee any additional enrichments being added to this array but it's still a great way to indicate and explain why. I'll update the PR accordingly.